File: Practicas/Practica2/PagoEmpleado.java
package Practicas.Practica2;
import java.util.Scanner;
/**
 *
 * @author Salas Chavez Jose de Jesus
 */
public class PagoEmpleado {

    public static double sueldoNormal(double sueldo, int hr) {
        return sueldo*hr;
    }
   
    public static double sueldoExtra(double sueldo, int hrx) {
        return 1.5*(sueldo*(hrx));
    }
   
   
   
    public static void totalSueldo(){
        Scanner scanner = new Scanner(System.in);
        double sueldo;
        int hr,hrx=0;
       
        System.out.print("Ingrese el número de horas trabajadas: ");
        hr = scanner.nextInt();
        System.out.print("Ingrese el sueldo por hora: ");
        sueldo = scanner.nextDouble();
       
        if(hr>40)
            hrx=hr-40;
       
        System.out.print("Paga regular: "+ sueldoNormal(sueldo,hr-hrx)+"\n");
        System.out.print("Numero de hrs extra: "+hrx+"\n");
        System.out.print("Paga extra: "+ sueldoExtra(sueldo,hrx)+"\n");
        System.out.print("Paga Total: "+(sueldoNormal(sueldo,hr-hrx)+sueldoExtra(sueldo,hrx))+"\n");
       
    }
   
    public static void main(String[] args) {
        System.out.println("-------------");
        System.out.println("Calculadora de sueldo");
        totalSueldo();
    }
   
}

File: Practicas/Practica3/PagoEmpleado.py
"""
Autor: Salas Chavez Jose de Jesus
"""
def sueldoNormal(sueldo,hr):
    return sueldo*hr

def sueldoExtra(sueldo,hrx):
    return 1.5*sueldo*hrx

def totalSueldo():
    hrx = 0
    hr=int(input("Ingrese el numero de horas trabajadas: "))
    sueldo = int(input("Ingrese el sueldo por hora: "))

    if hr>40:
        hrx=hr-40

    print(f"Paga regular: {sueldoNormal(sueldo,hr-hrx)}")
    print(f"Numero de hrs extra: {hrx}")
    print(f"Paga rextra: {sueldoExtra(sueldo,hrx)}")
    print(f"Paga Total: {sueldoNormal(sueldo,hr-hrx)+sueldoExtra(sueldo,hrx)}")
    return 0

def main():
    print("-------------")
    totalSueldo()


if __name__ == "__main__":
    main()